src/server.ts

```diff
@@ -11,20 +11,27 @@ import { api as unauthenticatedApi, views as unauthenticatedViews } from "./tool
 import { views as secretDjViews } from "./secret-dj/routers/views";
 import subdomain from "express-subdomain";
 import { adminViews } from "./toolbox/routers/views";
-import { z } from "zod";
 import emails from "./toolbox/storage/emails";
 import { parse } from "./parse";
 import { match, P } from "ts-pattern";
 import { Ok } from "./types/result";
 import session from "express-session";
-import { checkboxSchema } from "./util/checkboxSchema";
+import { settingsSchema } from "./schema";
 
 // TODO would be great to also serve a javascript client
 // TODO this whole system is a mess...
 
 const views = express()
 	.set("view engine", "ejs")
 	.use("/public", express.static("public"))
+	.use((req, res, next) => {
```

Comment:
nit: make this into middleware function that describes what this is doing/why

Reply:
yeah..........